jadx-gui/src/main/java/jadx/gui/treemodel/JMethod.java

```diff
@@ -12,22 +12,23 @@
 import jadx.core.dex.attributes.AFlag;
 import jadx.core.dex.info.AccessInfo;
 import jadx.core.dex.instructions.args.ArgType;
-import jadx.gui.ui.ContentPanel;
+import jadx.gui.ui.panel.ContentPanel;
 import jadx.gui.ui.TabbedPane;
 import jadx.gui.ui.codearea.ClassCodeContentPanel;
 import jadx.gui.utils.OverlayIcon;
 import jadx.gui.utils.UiUtils;
+import sun.security.mscapi.PRNG;
```

Comment:
Unused import which leads to compilation error.
Please format code before pushing PR, see [Code Formatting](https://github.com/skylot/jadx/wiki/Code-Formatting)

Reply:
I looked at this, the format of the code in this configuration does not work correctly in my IDEA (2021.2) Windows system. I do not know what to do...

Reply:
Just execute `gradle spotlessApply`.
> the format of the code in this configuration does not work correctly in my IDEA

 What kind of issue you have?

Reply:
dont fix problems files,